backend/fregepoc/settings.py

```diff
@@ -24,6 +24,8 @@
 DEBUG = os.environ.get("DJANGO_DEBUG", "true").lower() == "true"
 
 ALLOWED_HOSTS = [".localhost", "127.0.0.1"]
+if os.environ.get("BACKEND_HOSTNAME") is not None:
+    ALLOWED_HOSTS.append(os.environ.get("BACKEND_HOSTNAME"))
```

Comment:
There is no need to leave `.localhost` & `127.0.0.1` in `ALLOWED_HOSTS` if we're getting if from env vars. I would suggest to clean this up, and add `.localhost` as the default `BACKEND_HOSTNAME` in `.env.template`, as it will be re-configured in the production environment.